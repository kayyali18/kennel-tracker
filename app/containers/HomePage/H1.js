import styled from 'styled-components'

const H1 = styled.h1`
  text-align: center;
  font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
  color: #595959;
  font-size: 50px;
  font-weigth: 800;
`
export default H1
