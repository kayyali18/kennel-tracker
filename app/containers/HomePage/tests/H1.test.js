import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'

import H1 from '../H1'

describe('<H1 />', () => {
  it('should match snapshot', () => {
    const renderedComponent = renderer.create(<H1 />).toJSON()
    expect(renderedComponent).toMatchSnapshot()
  })

  it('should render an h1 tag', () => {
    const renderedComponent = renderer.create(<H1 />).toJSON()
    expect(renderedComponent.type).toEqual('h1')
  })
})
