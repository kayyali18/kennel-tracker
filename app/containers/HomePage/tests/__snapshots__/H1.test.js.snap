// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<H1 /> should match snapshot 1`] = `
.c0 {
  text-align: center;
  font-family: 'Helvetica Neue',Helvetica,Arial,sans-serif;
  color: black;
  font-size: 50px;
  margin: 0;
}

<h1
  className="c0"
/>
`;
